specification/datamigration/resource-manager/readme.md

```diff
@@ -50,6 +50,39 @@ input-file:
 - Microsoft.DataMigration/stable/2018-04-19/definitions/MigrationValidation.json
 ```
 
+### Tag: package-2018-07-15-preview
```

Comment:
You probably want to change line 31 as well, to put package-2018-07-15-preview as the default tag to generate

Reply:
Done.